terraform/tls-cert.tf

```diff
@@ -0,0 +1,46 @@
+resource "azurerm_key_vault_certificate" "automation_account" {
+  name         = local.cert_name
+  key_vault_id = data.azurerm_key_vault.cert_key_vault.id
+
+  certificate_policy {
+    issuer_parameters {
+      name = "Self"
+    }
+
+    key_properties {
+      exportable = true
+      key_size   = 2048
+      key_type   = "RSA"
+      reuse_key  = true
+    }
+
+    lifetime_action {
+      action {
+        action_type = "AutoRenew"
+      }
+
+      trigger {
+        days_before_expiry = 30
```

Comment:
nice but i assume the automation account needs updating too when it expires?

Reply:
Yeah I think so, haven't tested though. We'd need to apply tf again or have something like a runbook to update the automation account and sp

Reply:
thought so